examples/step-52/step-52.cc

```diff
@@ -354,11 +348,10 @@ namespace Step52
   //   - compute $z=\left(M-\tau \frac{\partial f}{\partial y}\right)^{-1} tmp =
   //   \left(M-\tau \frac{\partial f}{\partial y}\right)^{-1} My$
   //   - return z.
-  Vector<double> Diffusion::id_minus_tau_J_inverse(const double          time,
+  Vector<double> Diffusion::id_minus_tau_J_inverse(const double /*time*/,
```

Comment:
can you manually align the commented out variable to the level of the ones below? Or does clang-format undo it?

Reply:
No, I can't manually align the commented variable.